src/target/riscv/riscv-011.c

```diff
@@ -739,7 +739,7 @@ static int wait_for_debugint_clear(struct target *target, bool ignore_first)
 
 		if (!bits.interrupt)
 			return ERROR_OK;
-		if (time(NULL) - start > riscv_command_timeout_sec) {
+		if (time(NULL) - start > riscv_command_timeout_sec()) {
```

Comment:
Detail:

Since this is now a function (getter), not a plain variable, you may consider adding the "_get_" part to its name.

```suggestion
		if (time(NULL) - start > riscv_get_command_timeout_sec()) {
```

Reply:
Addressed.